priv/repo/migrations/20240525065205_update_user_session.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Dbservice.Repo.Migrations.UpdateUserSession do
+  use Ecto.Migration
+
+  def change do
+    alter table(:user_session) do
+      add :user_activity_type, :string
+      remove :session_occurrence_id
+      add :session_occurrence_id, references(:session_occurrence)
```

Comment:
What's the point of removing session occurrence and adding it again?

Reply:
It didnt have any reference, was storing it as a string. I could probably do update also.

Reply:
`session_occurrence_id` did had reference. Only user_id didn't had

Reply:
You're right, my bad, removed it.